vita3k/renderer/src/gl/renderer.cpp

```diff
@@ -486,7 +486,9 @@ void get_surface_data(GLContext &context, size_t width, size_t height, size_t st
         glReadPixels(0, 0, static_cast<GLsizei>(width), static_cast<GLsizei>(height), GL_RED, GL_UNSIGNED_BYTE, pixels);
         break;
     case SCE_GXM_COLOR_FORMAT_U2F10F10F10_ABGR:
+    case SCE_GXM_COLOR_FORMAT_F10F10F10U2_RGBA:
     case SCE_GXM_COLOR_FORMAT_U2U10U10U10_ABGR:
+    case SCE_GXM_COLOR_FORMAT_U10U10U10U2_RGBA:
         glReadPixels(0, 0, static_cast<GLsizei>(width), static_cast<GLsizei>(height), GL_RGBA, GL_UNSIGNED_INT_2_10_10_10_REV, pixels);
```

Comment:
If 2-10-10-10 is using GL_UNSIGNED_INT_2_10_10_10_REV then the new ones which are reversed should use GL_UNSIGNED_INT_10_10_10_2.

Reply:
Oh’ i have just used your commit’ is untouched

Reply:
you can re see it here
https://github.com/Zangetsu38/Vita3K/commit/1ca94c0cfcbd3a562b8dc0b4e95bb1370df9b661

Reply:
and cceck it for my version on other value, i have used what you said before and on my current build
https://github.com/Zangetsu38/Vita3K/commit/64eb69a94f377c9d08fb459c3a6c9219bdd03913

Reply:
i am just sur for current value here for U2F10F10F10_ABGR
![image](https://user-images.githubusercontent.com/5261759/121242454-de99b200-c89c-11eb-9b6e-4f7e80bcdb9a.png)
using other value get olor full broken